library/build.gradle

```diff
@@ -5,17 +5,17 @@ repositories {
 }
 
 dependencies {
-    compile 'com.android.support:support-v4:23.4.0'
-    compile 'com.android.support:support-annotations:23.4.0'
-    compile 'com.android.support:recyclerview-v7:23.4.0'
+    compile 'com.android.support:support-v4:26.+'
+    compile 'com.android.support:support-annotations:26.+'
+    compile 'com.android.support:recyclerview-v7:26.+'
```

Comment:
can you update to use the exact version without "+" and I'd be happy to merge this